schema.go

```diff
@@ -29,7 +29,6 @@ type Test struct {
 	Started   time.Time              `bson:"started"`
 	Ended     *time.Time             `bson:"ended"`
 	Info      map[string]interface{} `bson:"info"`
-	Failed    bool                   `bson:"failed"`
```

Comment:
I think we should probably keep this here even if we don't use it for legacy reasons? maybe safe to make it omit empty though